gpt4all-backend/llamamodel.cpp

```diff
@@ -36,6 +36,14 @@ namespace {
 const char *modelType_ = "LLaMA";
 }
 
+static void null_log_callback(enum ggml_log_level, const char*, void*) {
+}
```

Comment:
maybe we should keep the parameter names?
```suggestion
static void null_log_callback(enum ggml_log_level /*level*/, const char * /*text*/, void * /*user_data*/) {
}
```
(void casts would also work)